packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -159,7 +160,7 @@ export function NetworksAndSignersProvider(
         .then(async l1Network => {
 
           // Web3Provider is connected to an L1 network. We instantiate a provider for the L2 network.
-          const l2Provider = new JsonRpcProvider(rpcURLs[_selectedL2ChainId!])
+          const l2Provider = new StaticJsonRpcProvider(rpcURLs[_selectedL2ChainId!])
```

Comment:
This is cool! The docs for [StaticJsonRpcProvider](https://docs.ethers.io/v5/api/providers/jsonrpc-provider/#StaticJsonRpcProvider) state:

> This [Provider](https://docs.ethers.io/v5/api/providers/provider/) should only be used when it is known the network cannot change.

Not sure if we should worry about this, as we instantiate new providers / basically reload the app on network change. Still should spend some more time testing it, just in case.

Reply:
I think it is fair to expect the same `rpcURLs` won't serve another chain in our use case, unless it is something like a local node.